### ISSUE TITLE
Feature: recommending the mobile app when visiting browser experience on device

### ISSUE THREAD

**Issue by squigglybob** [Pray4Movement/prayer-global-porch#95]
Feature suggestion:

When a user hits the browser experience (not the prayer experience but any other part of the site)

We could send them a popup/toast suggesting that they download the app from their relevant app store.

This would help point people more strongly towards the app experience where we can communicate better with them.

**Comment by ChrisChasm** [comment]
Good idea. Could you itemize and maybe loom out your idea more in the comments here.

**Comment by squigglybob** [comment]
## Research

https://www.loom.com/share/67468183db02433b9e71e205ec3c6211

### ebay/amazon
I have a distinct feeling like when I went through the checkout process with ebay, they pushed their app on me as a better way to get notifications of deliveries etc.

Likewise with amazon, just browsing around didn't trigger anything, but I remember having the app pushed on me at some point, and probably during the checkout process.

I didn't experience the app being pushed on me this time, so mabye they have both backed off from this approach as it was too pushy.  I also remember a point where they offered money off first purchases with the app, so mabye they aren't targetting that on me anymore as I've already done it, or they have backed off from that too.

These 2 sites, (if they promote the app at checkout) are more promoting their app at a point that makes sense to the customer.

Site on landing: {acting coy, and not shouting out about their awesome app}
Site at checkout: "Why don't you try our app, you get these benefits (watch notifications, buy notifications etc.)"
User: "yeah that would be quite nice actually" --> downloads app


### pinterest/reddit

A bad experience that I just had with this, was with pinterest. I remember not long ago, they would quite heavily push the app experience on you, as would reddit.

These 2 really push their app on you when you first hit their site, and then when you try and read/look at things.

Personally, these 2 feel too pushy, like their main drive of the websites is to get you into the app:

Site on first landing: "Dude, don't go into our website, use our app instead!! No, seriously it's the app you want!"
User: "Hey, let me look at what I want to look at, and stop making me click buttons just to see your website!!"
User, broken by much badgering: "Look if it'll get you off my back, fine, i'll download the app! There, you happy now?"

## Brainstorming ideas for P.G.

I think we want to avoid annoying people with app promoting, so I'm personally against being as forward as pinterest/reddit
But... there could be a way that we make the app promotion a bit more visible

( That also makes me think, that we should remove any app promotion if the user is actually using the app )

### App promotion banners

The app promotion banner could go higher on the landing page, or we could add a thinner non full display height banner to promote it further up, like above the about section...

### App promotion popup/flyover

Thinking about the ebay/amazon way of doing things, makes me wonder if there are some optimal times when the app could be suggested as giving benefits to the user over the website experience.

The main benefit that I can think of is getting reminder notifications or group notifications.

This could be benifial to the challenge leader and the challenge participants.

So, one idea could be to promote the app when the user is starting/joining a challenge

**Comment by ChrisChasm** [comment]
Let's prioritize this after the deep linking.
